packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/mockapi.ts

```diff
@@ -61,7 +199,7 @@ Scenarios.Azure_ResourceManager_Models_Resources_TopLevelTrackedResources_action
         urgent: true,
       });
       return {
-        status: 204,
+        status: 200,
```

Comment:
Reason?

Reply:
This return status code has been rollbacked.